docs/en/verifier.md

```
@@ -666,34 +612,34 @@ Verifies the VP token.
   response: AuthorizationResponse,
-  options: VerifyPresentationOptions
-): Promise<VpTokenPayload>
+  transactionId: string,
+  options?: VerifyPresentationOptions
+): Promise<Record<string, VpTokenPayload[]>>
 ```
 
 **Parameters**:
-- `id`: Identifier of the Verifier ([VerifierClientId](#VerifierClientId))
+
 - `response`: Information used for verification ([Verifierauthorizationresponse](#Verifierauthorizationresponse))
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)